frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/UpdateProfileHandler.java

```diff
@@ -88,7 +88,6 @@ public APIGatewayProxyResponseEvent handleRequest(
         String clientId;
 
         if (session.isEmpty()) {
-            LOGGER.info("Session is empty.");
```

Comment:
Not needed as the SessionService will already have output a suitable error message to the log.